meta-networking/recipes-connectivity/autossh/autossh_1.4g.bb

```diff
@@ -0,0 +1,30 @@
+DESCRIPTION = "autossh is a program to start a copy of ssh and monitor it, restarting it as necessary should it die or stop passing traffic"
+HOMEPAGE = "https://www.harding.motd.ca/autossh/"
+
+LICENSE = "BSD"
```

Comment:
can you see if its specific BSD license e.g. BSD-3-Clause

Reply:
I'm not sure where I got BSD as the license.  According to the package, it is licensed as "Distributable" and according to the Free Software Directory autossh is licensed under SimplePermissiveNoNonWarranty.  Not sure what license to use from licenses.conf...